### PR TITLE
Transaction cost routing

### PR DESCRIPTION
**Pull request by dekz**
Adjust bridge slippage to 1%.
Upgrade Asset-swapper to account for transaction cost routing.
Set reasonable defaults by source.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -89,10 +89,19 @@ export const ASSET_SWAPPER_MARKET_ORDERS_OPTS: Partial<SwapQuoteRequestOpts> = {
...
     sampleDistributionBase: 1.05,
+    fees: {
+        [ERC20BridgeSource.Uniswap]: new BigNumber(2.5e5),
+        [ERC20BridgeSource.Native]: new BigNumber(3e5),
+        [ERC20BridgeSource.CurveUsdcDai]: new BigNumber(4e5),
+        [ERC20BridgeSource.Eth2Dai]: new BigNumber(5e5),
+        [ERC20BridgeSource.CurveUsdcDaiUsdt]: new BigNumber(5e5),
+        [ERC20BridgeSource.CurveUsdcDaiUsdtTusd]: new BigNumber(8e5),
+        [ERC20BridgeSource.Kyber]: new BigNumber(8e5),
+    },
```
Comment (fragosti) [inline src/config.ts]: nit: it should be possible to type this such that this doesn't compile unless this map is exhaustive.

**Comment by fragosti** [comment]
Are you going to make any modifications to the `slippagePercentage` ?

**Comment by dekz** [comment]
> Are you going to make any modifications to the `slippagePercentage` ?

Yes, though in a separate PR.

**Comment by dekz** [comment]
deploy staging

**Pull request merged**